src/index.html

```diff
@@ -13,7 +13,33 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-
+  <div id="qa-block" class="block-blue block">
+    <div class="block-orange block">
```

Comment:
will be better to name block-outer, block-intermediate, etc. instead of using colors

Reply:
rename block